examples/plotting/line_on_off.py

```diff
@@ -1,5 +1,11 @@
-""" Example demonstrating turning lines on and off - with JS only
-
+""" Example demonstrating turning lines on and off - with JS only.
+It involves checking and unchecking the checkboxes representing
+the plotted lines to turn the lines on/off.
+
+.. bokeh-example-metadata::
+    :apis: bokeh.plotting.figure.line, bokeh.layouts.row, bokeh.models.CustomJS
+    :refs: :ref:`ug_basic_lines_line_multiple`,:ref:`_ug_interaction_widgets_examples_checkboxgroup`,:ref:`ug_interaction_js_callbacks`
```

Comment:
one ref slightly off:
```suggestion
    :refs: :ref:`ug_basic_lines_multi`,:ref:`_ug_interaction_widgets_examples_checkboxgroup`,:ref:`ug_interaction_js_callbacks`
```

Reply:
Done.